triscord/trello.py

```diff
@@ -108,8 +110,15 @@ def actions(self):
         # Reverse actions order to sort them chronologically
         actions = response.json()
         actions.reverse()
+        
+        aliases = {}
+        if 'Aliases' in settings.CONFIG.sections():
+            aliases = settings.CONFIG['Aliases']
 
         for action in actions:
+            trello_username = action['memberCreator']['username']
+            if trello_username in aliases:
+                action['memberCreator']['username'] = aliases[trello_username]
```

Comment:
Sounds good. You'll want to replace `action['display']['entities']['memberCreator']['username']` as well as `action['member']['username']` because the Trello API is silly like that.

Make sure to check out all the action formatters to see what data is used where.